packages/notifi-react-card/lib/components/AlertHistory/HealthValueOverThresholdEventRenderer.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import { AnnouncementIcon } from '../../assets/AnnouncementIcon';
+import { AlertNotificationRow } from './AlertNotificationRow';
+
+type HealthValueOverThresholdEventRendererProps = Readonly<{
+  createdDate: string;
+  name: string | undefined;
+  threshold: string | undefined;
+  notificationTitle: string;
+  handleAlertEntrySelection: () => void;
+  healthCheckIcon?: JSX.Element;
+}>;
+
+export const HealthValueOverThresholdEventRenderer: React.FC<
+  HealthValueOverThresholdEventRendererProps
+> = ({
+  name,
+  threshold,
+  createdDate,
+  notificationTitle,
+  handleAlertEntrySelection,
+  healthCheckIcon,
+}) => {
+  const getDefaultIcon = () => {
+    return healthCheckIcon || <AnnouncementIcon />;
```

Comment:
any idea whether we can use a different icon as default? Not a big deal.

Reply:
updated icon for health check
![Screenshot 2023-01-05 at 12 26 06 PM](https://user-images.githubusercontent.com/26240001/210873685-6ab4ada7-09b4-47e3-afea-61cdde3763aa.png)